CHANGELOG.md

```diff
@@ -4,6 +4,18 @@ All notable changes to this project will be documented in this file. Dates are d
 
 #### [Unreleased](https://github.com/googlemaps/android-maps-utils/compare/1.0.0...HEAD)
 
+#### [1.0.1](https://github.com/googlemaps/android-maps-utils/compare/1.0.0...1.0.1)
+
+> 18 March 2020
+
+### Features
+* Expose PolyUtil.DEFAULT_TOLERANCE ([46cbac5](https://github.com/googlemaps/android-maps-utils/commit/46cbac53883d0c5a99492b005b18a3ea558737b1))
+
+### BREAKING CHANGES
```

Comment:
This BREAKING CHANGES section shouldn't be here, right? That's specific to v1.0.0.

Reply:
Wups! Did this too quick. 😓 Will update.

Reply:
Done d8c7ae4